pyccel/stdlib/gFTL_functions/Map_extensions.inc

```diff
@@ -0,0 +1,48 @@
+#include <map/header.inc>
+
+contains
+
+#define __IDENTITY(x) x
+#define __guard __map_guard
+#include "parameters/Key/copy_map_Key_to_internal_Key.inc"
+#include "parameters/T/copy_map_T_to_internal_T.inc"
+#include "parameters/Key/define_derived_macros.inc"
+#include "parameters/T/define_derived_macros.inc"
+
+  function __IDENTITY(Map)_pop(my_map, key) result(value)
```

Comment:
Is the functions' indentation given on purpose, or by accident?

Reply:
On purpose to match the usual indentation of a function in a module and what is done in similar files, e.g.
https://github.com/pyccel/pyccel/blob/devel/pyccel%2Fstdlib%2FgFTL_functions%2FVector_extensions.inc

Reply:
Sounds good!